src/erlang_ls.erl

```diff
@@ -4,6 +4,15 @@
 
 -spec main([any()]) -> ok.
 main([Port]) ->
+  init_node_name(),
   application:set_env(erlang_ls, port, list_to_integer(Port)),
   application:ensure_all_started(erlang_ls),
+  lager:info("Started erlang_ls server ~p", [node()]),
   receive _ -> ok end.
+
+-spec init_node_name() -> ok.
+init_node_name() ->
+  Name     = "erlang_ls_" ++ integer_to_list(rand:uniform(16#FFFFFFFFF)),
+  NodeName = list_to_atom(Name),
+  net_kernel:start([NodeName, shortnames]),
```

Comment:
This requires EPMD to be already started or it will fail, right?

Reply:
See for example: https://github.com/robertoaloi/ansible-nodetool/blob/master/nodetool#L123

Reply:
Yup, but AFAIK there is no way to ensure EPMD is started, other than doing some dirty hack with `os:cmd/1`. Right?

Reply:
Not that I know. RabbitMQ does the same: https://github.com/rabbitmq/rabbitmq-common/blob/master/src/rabbit_nodes_common.erl#L73

Reply:
Updated the PR.